packages/core/src/odata-v2/request-builder/get-all-request-builder.spec.ts

```diff
@@ -39,6 +42,19 @@ describe('GetAllRequestBuilder', () => {
         .url(defaultDestination);
       expect(actual).toBe(expected);
     });
+
+    it('is built correctly select in expand', async () => {
```

Comment:
I am not sure that I fully understand the meaning of this, can you rephrase it to be a bit clearer?